<map id="Mezzanine::NetworkPermissionException" name="Mezzanine::NetworkPermissionException">
<area shape="rect" id="node2" href="$classMezzanine_1_1NetworkException.html" title="Thrown when something unknown causes network IO to fail. " alt="" coords="8,237,208,267"/>
<area shape="rect" id="node4" href="$classMezzanine_1_1IOException.html" title="Thrown when there was an issue with IO but very little was known about it. " alt="" coords="25,160,191,189"/>
<area shape="rect" id="node6" href="$classMezzanine_1_1Exception.html" title="This is the exception thrown by most Mezzanine system that can throw exceptions. " alt="" coords="33,83,183,112"/>
<area shape="rect" id="node8" title="STL class. " alt="" coords="56,5,160,35"/>
</map>
